src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -608,6 +637,7 @@ def num_timesteps(self):
     def __call__(
         self,
         prompt: Union[str, List[str]] = None,
+        image_prompt: PipelineImageInput = None,
```

Comment:
Can we move this further down (maybe below `negative_prompt_embeds`) and call it `ip_adapter_image`? It should be clear to the user that 

- a) It's an optional (less important) call input
- b) It's related to the IP adapter only

Reply:
Done.